tests/test_glue.py

```diff
@@ -216,6 +216,58 @@ def test_glue_job_untag(self):
         self.assertEqual(tags.get('Tags'), {})
         self.assertEqual(len(resources), 1)
 
+    def test_glue_crawler_tag(self):
+        session_factory = self.replay_flight_data("test_crawler_tags")
+        client = session_factory().client("glue")
+
+        tags = client.get_tags(ResourceArn='arn:aws:glue:us-east-1:644160558196:crawler/test')
+        self.assertEqual(tags.get('Tags'), {})
+
+        policy = {
+            'name': 'test',
+            'resource': 'glue-crawler',
+            'actions': [
+                {
+                    'type': 'tag',
+                    'key': 'abcd',
+                    'value': 'xyz'
+                },
+            ]
+        }
+        p = self.load_policy(
+            policy,
+            config={'account_id': '644160558196'},
+            session_factory=session_factory)
+
+        resources = p.run()
+        arn = p.resource_manager.generate_arn(resources[0]['Name'])
+        self.assertEqual(arn, 'arn:aws:glue:us-east-1:644160558196:crawler/test')
+        tags = client.get_tags(ResourceArn=arn)
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(tags.get('Tags'), {'abcd': 'xyz'})
+
+    def test_glue_crawler_untag(self):
+        session_factory = self.replay_flight_data("test_glue_crawler_untag")
+
+        policy = {
```

Comment:
there's no assertion here that the tags were on the resource previously

Reply:
added 'filters' to check tags exist before